README.md

```diff
@@ -108,10 +108,11 @@ The executable was built in Kubuntu 22.04 LTS so there is a chance that it will
 
 #### Install via Flatpak
 
-Thanks to [Massi aka Ergosum](https://github.com/mbugni), Doom Runner can now be installed via [Flatpak](https://flatpak.org/) from this package TODO: link, using the following commands:
-```
-TODO
-```
+Doom Runner can be installed via [Flatpak](https://flatpak.org/) from this link:
+
+[![Download Doom Runner](https://dl.flathub.org/assets/badges/flathub-badge-en.png 'Download Doom Runner')](https://flathub.org/apps/io.github.Youda008.DoomRunner)
```

Comment:
Was it intended to display this big button or is it accidental?
![obrazek](https://github.com/Youda008/DoomRunner/assets/20306543/b7fbf35f-1477-4832-a151-d8ba60cdbaa3)

Reply:
It's not accidental, is an [official badge](https://flathub.org/badges). A white version is available too.

Reply:
ok, good